README.md

```diff
@@ -273,6 +274,49 @@ your browser.
     'Client secret'.
 
 
+### 3.2. Facebook OAuth
+
+ 1. Go to https://developers.facebook.com/ and create an app.
+
+ 2. Add a Product 'Facebook Login' and select platform 'Web'. Enter 'Site URL' `https://{domain}/`.
+
+ 3. Go to 'Facebook Login' / 'Settings' and add 'Valid OAuth Redirect URIs':
+
+    	https://{domain}/accounts/facebook/login/callback/
+    	https://{domain}/sk/accounts/facebook/login/callback/
+    	https://{domain}/en/accounts/facebook/login/callback/
+
+    Where `{domain}` is your production domain or `127.0.0.1:8000` if in local development mode.
+    'Site URL' in form `127.0.0.1:8000` is invalid use `localhost:8000` instead.
+
+To use Facebook Login, you need to use https. Setup default protocol in `settings/common.py`.
+
+	ACCOUNT_DEFAULT_HTTP_PROTOCOL = u'https'
+
+To run a server with https you can use
+[RunServerPlus](https://django-extensions.readthedocs.org/en/latest/runserver_plus.html) from
+[django-extensions](https://github.com/django-extensions/django-extensions). To install run:
+
+	$ pip install django-extensions==1.0.3
+	$ pip install Werkzeug==0.16.1
+	$ pip install pyOpenSSL==19.1.0
+
+Add it to `INSTALLED_APPS` in `settings/common.py` file:
+
+	INSTALLED_APPS = (
+	...
+	'django_extensions',
+	...
+	)
+
+Run the server in ssl mode with:
+
+	./manage.py runserver_plus --cert /tmp/cert
```

Comment:
@viliambalaz Nezafungovalo. Netusis, kde mohol byt problem. Resp. na co som zabudol?

1. zmazal som `env` a vytvoril nanovo s `python setup.py` s modom 1, nech mame ciste `env`
2. `env/bin/pip install django-extensions==1.0.3` zbehlo
3. `env/bin/pip install Werkzeug==0.16.1` zbehlo
4. `env/bin/pip install pyOpenSSL==19.1.0` padlo na chybu: "cryptography requires setuptools 18.5 or newer, please upgrade to a"
5. tak som upgradol setuptools: `env/bin/pip install setuptools==18.5` zbehlo
6. znovu skusam `env/bin/pip install pyOpenSSL==19.1.0`, padlo na chybu: "No package 'libffi' found"
7. tak nainstalujem `sudo apt-get install libffi-dev`
8. znovu skusam `env/bin/pip uninstall pyOpenSSL==19.1.0`, zbehlo.

Doplnim `ACCOUNT_DEFAULT_HTTP_PROTOCOL = u'https'` a `django_extensions` do `chcemvediet/settings/common.py` a spustim: `env/bin/python manage.py runserver_plus --cert /tmp/cert`

Padlo na chybe: "CommandError: Python OpenSSL Library is required to use runserver_plus with ssl support. Install via pip (pip install pyOpenSSL)."

Reply:
@martinmacko47 S touto chybou som sa nestretol. Pravdepodobne bude problem len vo verziach package-ov. Moje verzie su: 
```
setuptools              41.0.1
libffi-dev/xenial 3.2.1-4 amd64
```
Pri instalovani `django_extensions` a `Werkzeug` som si nainstaloval najnovsiu verziu a postupne som ich downgrade-oval az pokial to cele zbehlo. Mozno pomoze aj downgrade `pyOpenSSL`. Skus vyskusat, potom by to bolo dobre spisat tiez do dokumentacie.

Reply:
@viliambalaz Super, podarilo sa mi to rozbehat. Pouzivame verziu Djanga radovo z roku 2003. Package `django-extensions==1.0.3` je tiez z roku 2003. Tak som pouzil aj ostatne kniznice z toho isteho roku: `Werkzeug==0.9` a `pyOpenSSL==0.13`.

Zafungovalo. Testovaciu FB app sa mi podarilo spravit podla postupu. Takze pls zapracuj pripomienky a FB mame vyrieseny.

Reply:
:ok: